SPECS-EXTENDED/libomxil-bellagio/libomxil-bellagio.spec

```diff
@@ -0,0 +1,257 @@
+Summary:        OpenMAX Integration Layer
+Name:           libomxil-bellagio
+Version:        0.9.3
+Release:        29%{?dist}
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://omxil.sourceforge.net
+Source0:        http://downloads.sourceforge.net/omxil/%{name}-%{version}.tar.gz
+#https://sourceforge.net/tracker/?func=detail&aid=3477869&group_id=160680&atid=816817
+Patch0:         libomxil-bellagio-0.9.3-fix_Werror.patch
+Patch1:         libomxil-bellagio-0.9.3-unused.patch
+#https://sourceforge.net/tracker/?func=detail&aid=3477871&group_id=160680&atid=816817
+Patch2:         libomxil-bellagio-0.9.3-nodoc.patch
+Patch3:         http://git.buildroot.net/buildroot/plain/package/multimedia/bellagio/bellagio-0.9.3-dynamicloader-linking.patch
+Patch4:         http://git.buildroot.net/buildroot/plain/package/multimedia/bellagio/bellagio-0.9.3-parallel-build.patch
+Patch5:         http://git.buildroot.net/buildroot/plain/package/multimedia/bellagio/bellagio-0.9.3-segfault-on-removeFromWaitResource.patch
+Patch6:         omxil_version.patch
+Patch7:         libomxil-bellagio-0.9.3-memcpy.patch
+Patch8:         libomxil-bellagio-0.9.3-valgrind_register.patch
+%define         _legacy_common_support 1
+BuildRequires:  doxygen
+BuildRequires:  gcc-c++
+BuildRequires:  libtool
+BuildRequires:  make
+
+%description
+The OpenMAX IL API defines a standardized media component interface to
+enable developers and platform providers to integrate and communicate
+with multimedia codecs implemented in hardware or software.
+
+The libomxil shared library implements the OpenMAX IL Core functionalities.
+Three dynamically loadable components are also included: OMX alsa sink
+component, OMX mp3,aac,ogg decoder component and OMX volume control component.
+
+%package        devel
+Summary:        Development files for %{name}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description    devel
+The %{name}-devel package contains libraries and header files for
+developing applications that use %{name}.
+
+%package        test
+Summary:        Test cases for %{name}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description    test
+The %{name}-test package contains binaries for testing %{name}.
+
+%prep
+%setup -q
```

Comment:
Consider autosetup

Reply:
@neha170, Thank you for the approval. Few patches are supplied with -p1 and few patches are not. This is the reason %autosetup does not work here. I have even tried replacing it with %autosetup but it is failing in applying the patches and hence keeping the changes as is.